src/gatsby/rssFeedPlugin.mjs

```diff
@@ -154,6 +112,6 @@ export default [{
       }
     `,
 
-    feeds: [blogFeed, changelogFeed, blogChangelogFeed],
+    feeds: [blogFeed, blogChangelogFeed],
```

Comment:
Does this remove the rss feed for the changelog: https://roadie.io/blog-changelog/rss.xml

It is used in the roadie application to populate the changelog in the app.

Reply:
<img width="1288" height="522" alt="image" src="https://github.com/user-attachments/assets/898c4108-8dc1-49c6-b834-f65ce59020e7" />